scripts/deploy-helloWorld.ts

```diff
@@ -0,0 +1,18 @@
+import { ethers } from 'hardhat';
+
+// deploy function
+export const deployHelloWorld = async () => {
+  async function main() {
+    const HelloWorld = await ethers.getContractFactory('HelloWorld');
+    const helloWorld = await HelloWorld.deploy();
+
+    await helloWorld.deployed();
+
+    console.log(`🚀 ~ Hello World contract deployed to ${helloWorld.address}`);
+  }
+
+  main().catch(error => {
+    console.error(error);
+    process.exitCode = 1;
+  });
+};
```

Comment:
remove main and handle promises with try/catch

Reply:
Done.